src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+type Prop = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Prop> = ({ todo }) => {
+  const { title, id, completed, user } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={cn('TodoInfo', { 'TodoInfo--completed': completed })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      {user ? <UserInfo user={user} /> : <></>}
```

Comment:
```suggestion
      {user && <UserInfo user={user} />}
```
You don't need to use the ternary operator if you return nothing in the second case